### DISCUSSION TITLE
semantic-release trying to publish same beta version

### DISCUSSION THREAD

**Discussion by thatkookooguy** [semantic-release/semantic-release#2213] [Q&A] [answered]
for some reason, my semantic release github action started getting stuck on my beta releases on version `v1.0.0-beta.10`.

checkout the output here: https://github.com/Kibibit/configit/actions/runs/1396454571

repository: https://github.com/Kibibit/configit

Tried pushing some tags of more advanced versions to force a release with a newer number (`v1.0.0-beta.12`)
But the release pipeline still tries to release `v1.0.0-beta.10`

any idea why this happened?
or how to fix this and get out of this lock?

thank you 🙏🏽

**Comment by travi** [comment, marked as answer]
give [these steps](https://semantic-release.gitbook.io/semantic-release/support/troubleshooting#release-not-found-release-branch-after-git-push-force) a try. the git notes are likely not matching your tags

**Comment by thatkookooguy** [comment]
@travi YES! fixed it! thank you 🙏🏽 🙏🏽 🙏🏽
